### DISCUSSION TITLE
cant deinstall xformers 0.0.16

### DISCUSSION THREAD

**Discussion by Gugu32** [facebookresearch/xformers#739] [Q&A]
Hello!
im quite new to this so any help appreciated! 

I tried to install xformers, since i got a batch-related error "RuntimeError: No executable batch size found, reached zero."
Therefore i followed an tutorial (https://github.com/AUTOMATIC1111/stable-diffusion-webui/wiki/Xformers) but couldnt build xformers in the end. Then i found out, i can just pip install xformers. Well, it installed the version 16, which isnt compatible with dreambooth. 
So i followed this: https://github.com/facebookresearch/xformers/discussions/717#discussioncomment-5729377, quite successful. 
Got xformers 19 installed. 
The only issue: when starting up the webui.bat i still get: xformers version 0.0.16rc425 installed.
I tried pip deinstall xformers when cd on directory c and f (c=hd which contains windows and stuff, f=hd which contains stable diffusion webui ). it could not find any (after deinstalling the v19.
i also tried --xformers --reinstall-xformers as args in the webui-user.bat. still nothing.
Anyone can help me out?

**Comment by danthe3rd** [comment]
Hi,
I'm not sure we can help with that, as this is more dependant on how your stable diffusion webui is setup. Maybe try posting an issue [there](https://github.com/AUTOMATIC1111/stable-diffusion-webui/issues)?
